src/combinator.rs

```diff
@@ -1693,7 +1692,6 @@ pub struct OrNot<A> {
     pub(crate) parser: A,
 }
 
-// TODO: Maybe implement `IterParser` too?
```

Comment:
There doesn't appear to be a reasonable way to implement `IterParser` for `OrNot` - if you have one in mind I can try that instead.

Reply:
That's fair enough